lib/feeder_importer.rb

```diff
@@ -0,0 +1,95 @@
+require 'prx_access'
+require 'addressable/uri'
+
+class FeederImporter
+  include PRXAccess
+  include Announce::Publisher
+
+  attr_accessor :account_id, :user_id, :feeder_podcast_url
+  attr_accessor :podcast, :series, :template, :distribution
```

Comment:
assume these two `attr_accessor` calls are kept separate for organization's sake?

Reply:
yes, and line length would be > 100 if they were all on one line

Reply:
👍